extra/index.json

```diff
@@ -1317,6 +1317,38 @@
 			},
 			"icon": "ted",
 			"type": "charter"
+		},
+		{
+			"ids": [ "santarockstar", "santarockstarhd", "santa" ],
+			"names": {
+				"en-US": "Santa Rockstar"
+			},
+			"icon": "santahd",
+			"type": "game"
+		},
+		{
+			"ids": [ "Mickelräven", "Mickelraven" ],
+			"names": {
+				"en-US": "Mickelräven"
+			},
+			"icon": "Mickelraven",
```

Comment:
Icon file names should always be lowercase, as it could cause problems on OS's where case matters. Make sure to update the `.png` file's name as well. You may need to remove it, and add it back in separate commits because of this.